### PR TITLE
SUMO-242752 | Changes in IIS otel app to add metric collection and dashboards

### PR DESCRIPTION
**Pull request by sumoanema**
Changes in IIS otel app to add metric collection and dashboards.

## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md [hunk 1]
```diff
@@ -11,19 +11,23 @@ import TabItem from '@theme/TabItem';
...
 
 IIS logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
 
+IIS metrics are sent to Sumo Logic through OpenTelemetry through [windowsperfcountersreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowsperfcountersreceiver). This receiver, captures the configured system, application, or custom performance counter data from the Windows registry using the [PDH interface](https://docs.microsoft.com/en-us/windows/win32/perfctrs/using-the-pdh-functions-to-consume-counter-data). It is based on the [Telegraf Windows Performance Counters Input Plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/win_perf_counters).
+
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/IIS-OpenTelemetry/IIS-Schematics.png' alt="Schematics" />
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:20]: Make sure the image is updated with Metrics information.

File: docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md [hunk 2]
```diff
@@ -235,3 +287,34 @@ The **IIS - Visitor Locations** dashboard provides a high-level view of Nginx vi
...
+
+### ASP.NET Applications
+
+The **IIS - ASP.NET Applications** dashboard provides a high-level view of the ASP.NET application performance counters. This dashboard helps you to monitor compilations, errors, cache, requests executing, requests in application queue, pipeline instance count, and output cache.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/IIS-OpenTelemetry/IIS-ASP.NET-Applications.png' alt="IIS-ASP.NET-Applications" />
+
+### Cache Performance
+
+The **IIS - Cache Performance** dashboard provides a high-level view of the the Web Service Cache Counters object includes cache counters specific to the World Wide Web Publishing Service. This dashboard helps you to monitor the output cahce, cache memory, file cache, and URI cache.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:311]:
```suggestion
The **IIS - Cache Performance** dashboard provides a high-level view of the the Web Service Cache Counters object includes cache counters specific to the World Wide Web Publishing Service. This dashboard helps you to monitor the output cache, cache memory, file cache, and URI cache.
```

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md [hunk 3]
```diff
@@ -11,19 +11,23 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/microsoft-azure/microsoft_iis_10.png')} alt="thumbnail icon" width="130"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-The [Internet Information Services](https://learn.microsoft.com/en-gb/iis/get-started/introduction-to-iis/introduction-to-iis-architecture) (IIS) 10 app is a logs app that helps you monitor your IIS web servers' availability. Preconfigured dashboards provide insight into application pools, ASP.NET applications, requests, latency, visitor locations, visitor access types, traffic patterns, errors, web server operations, and access from known malicious sources.
+The [Internet Information Services](https://learn.microsoft.com/en-gb/iis/get-started/introduction-to-iis/introduction-to-iis-architecture) (IIS) 10 app is a logs and metrics app that helps you monitor your IIS web servers' availability. Preconfigured dashboards and searches provide insight into application pools, ASP.NET applications, requests, latency, visitor locations, visitor access types, traffic patterns, errors, web server operations, and access from known malicious sources.
```
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:14]:
```suggestion
The [Internet Information Services](https://learn.microsoft.com/en-gb/iis/get-started/introduction-to-iis/introduction-to-iis-architecture) (IIS) 10 app is a logs and metrics app designed to monitor the availability and performance of your IIS web servers. Preconfigured dashboards and searches provide insight into application pools, ASP.NET applications, requests, latency, visitor locations, visitor access types, traffic patterns, errors, web server operations, and access from known malicious sources.
```

Comment (JV0812) [reply docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:14]: @sumoanema Can you also add the new dashboard information in this description.

File: docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md [hunk 4]
```diff
@@ -235,3 +287,34 @@ The **IIS - Visitor Locations** dashboard provides a high-level view of Nginx vi
...
+
+### Application Pool
+
+The **IIS - Application Pool** dashboard provides a high-level view of Application Pool State, Information and Worker Process Metrics.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/IIS-OpenTelemetry/IIS-Application-Pool.png' alt="IIS-Application-Pool" />
+
+### ASP.NET
+
+The **IIS - ASP.NET** dashboard provides a high-level view of the ASP.NET global performance counters. This dashboard helps you to analyse the state server sessions, monitor applications information, and understand the request execution and wait time.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:299]:
```suggestion
The **IIS - ASP.NET** dashboard provides a high-level view of the ASP.NET global performance counters. This dashboard helps you to analyse the state server sessions, monitor applications performance, and understand the request execution and wait time.
```

File: docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md (hunk 3 above)
Comment (sumoanema) [reply docs/integrations/web-servers/opentelemetry/iis-10-opentelemetry.md:14]: This description already has the info for all the dashboard which I have added now.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**